alternator/stats.cc

```diff
@@ -98,6 +96,10 @@ stats::stats() : api_operations{} {
                     seastar::metrics::description("number of rows read and matched during filtering operations")),
             seastar::metrics::make_total_operations("filtered_rows_dropped_total", [this] { return cql_stats.filtered_rows_read_total - cql_stats.filtered_rows_matched_total; },
                     seastar::metrics::description("number of rows read and dropped during filtering operations")),
+                    seastar::metrics::make_counter("item_count", seastar::metrics::description("The total number of items processed across all batches"),{op("BatchWriteItem")},
```

Comment:
You had another comment about the name, I don't have a strong opinion about it. I can remove the batch from the description, but then, how do you describe that it's part of a collection of items (batch or batch-like)?

If you can suggest a name that you feel comfortable with it would be great.